packages/components/src/components/table/table-header.tsx

```diff
@@ -1,16 +1,15 @@
 import React from 'react';
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
 
-const Header = ({ children, className }) => (
+type THeader = {
+    children: React.ReactNode;
```

Comment:
is there a reason that you didn't use `React.PropsWithChildren` here?

Reply:
because children are required here.  and children in PropsWithChildren are optional

Reply:
Got it.